src/OrchardCore.Modules/OrchardCore.Redis/HealthChecks/Startup.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Modules;
+
+namespace OrchardCore.Redis.HealthChecks;
+
+[RequireFeatures("OrchardCore.HealthChecks")]
+public class Startup : StartupBase
+{
+    public override int Order => 100;
```

Comment:
Add a comment on why this is necessary, and what's the reason for `100` in particular.

Reply:
I might need to check this one, I remember this might be necessary to run the health check early

Reply:
So, also document what's the reason for 100 in particular. This is a magic number; can it be 90? Can it be 110? Why not 1000?

Reply:
Could be, no particular reason for using `10` but usually we're using 100 or more to avoid start ordering issue

Reply:
OK, so please document it.

Reply:
BTW there are many places where we use 1000 for instance without docs :)

Reply:
And those should have been documented as well :). Having less-than-ideal pieces of code currently doesn't mean that we should keep writing the same.